bin.js

```diff
@@ -121,12 +136,12 @@ const readStdin = () => new Promise(resolve => {
   });
 });
 
-const prep = async ({ input, format = 'jpg' }) => {
+const prep = async ({ input, format = 'jpg', quality = 1 }) => {
   const buffer = input === '-' ?
     await readStdin() :
     await promisify(fs.readFile)(path.resolve('.', input));
 
-  const results = await convert.all({ buffer, format: FORMAT[format], quality: 1 });
+  const results = await convert.all({ buffer, format: FORMAT[format], quality: quality });
```

Comment:
```suggestion
  const results = await convert.all({ buffer, format: FORMAT[format], quality });
```

This is a good note for me that I actually forgot to add this rule to eslint. I will do that separately.